File: patterns/C++/pattern2.cpp
#include<bits/stdc++.h>
using namespace std;

void pattern2(int n){
    for(int i=0; i<n; i++){
        for(int j=0; j<i+1; j++){
            cout << "* ";
        }
        cout << "\n";
    }
}

int main(){
    int n;
    cout << "Enter the number of rows: ";
    cin >> n;
    cout << "\nHere is the generated pattern\n";
    pattern2(n);

}

File: patterns/Python/pattern2.py
"""
Generate the following pattern
*
* *
* * *
* * * *
* * * * *
"""

def pattern2(n):
    for i in range(n):
        for j in range(i+1):
            print("* ", end="")
        print("\n")
        

def main():
    i = int(input("Enter the number of rows: "))
    print("Here is the generated pattern\n")
    pattern2(i)
    
if __name__ == "__main__":
    main()